src/runtime/node/console/index.ts

```diff
@@ -0,0 +1,230 @@
+import type console from "node:console";
+import { createNotImplementedError } from "../../_internal/utils";
+import mock from "../../mock/proxy";
+
+export const assert: typeof console.assert = (
+  ...args: Parameters<(typeof globalThis)["console"]["assert"]>
+) => {
+  if (globalThis.console && globalThis.console.assert) {
+    return globalThis.console.assert(...args);
+  }
```

Comment:
you could simplify this a bit with safe navigation operator:

```suggestion
  if (globalThis.console?.assert) {
    return globalThis.console.assert(...args);
  }
```

and use this pattern throughout this whole file